[2018-05-14 23:50:17,249] launcher  INFO    SCOOP 0.7 1.1 on linux using Python 3.5.2 (default, Nov 23 2017, 16:37:01) [GCC 5.4.0 20160609], API: 1013
[2018-05-14 23:50:17,249] launcher  INFO    Deploying 8 worker(s) over 1 host(s).
[2018-05-14 23:50:17,249] launcher  INFO    Worker distribution: 
[2018-05-14 23:50:17,249] launcher  INFO       127.0.0.1:	7 + origin
2018-05-14 23:50:17,674 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,674 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,674 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,674 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,674 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,675 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,675 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,675 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,675 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,682 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,683 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,683 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,683 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,684 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,684 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,690 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,690 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,691 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,691 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,691 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,692 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,730 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,730 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,730 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,730 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,730 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,730 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,734 parameterhelper INFO     Loading config for dimensionality as a percentage of the reservoir size.
2018-05-14 23:50:17,734 parameterhelper INFO     Config: ./configs/user/bayeridge_pca (overload)
2018-05-14 23:50:17,735 root            INFO     Loading data (./data/Rainfall)
2018-05-14 23:50:17,743 root            INFO     Initializing genetic algorithm
2018-05-14 23:50:17,743 root            INFO     Initializing genetic algorithm
2018-05-14 23:50:17,745 root            INFO     Running GA optimization
2018-05-14 23:50:17,745 root            INFO     Running GA optimization
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/bootstrap/__main__.py", line 302, in <module>
    b.main()
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/bootstrap/__main__.py", line 92, in main
    self.run()
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/bootstrap/__main__.py", line 290, in run
    futures_startup()
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/bootstrap/__main__.py", line 271, in futures_startup
    run_name="__main__"
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/futures.py", line 64, in _startup
    result = _controller.switch(rootFuture, *args, **kargs)
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/_control.py", line 253, in runController
    raise future.exceptionValue
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/_control.py", line 127, in runFuture
    future.resultValue = future.callable(*future.args, **future.kargs)
  File "/usr/lib/python3.5/runpy.py", line 254, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/usr/lib/python3.5/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "./genoptesn.py", line 388, in <module>
    main()
  File "./genoptesn.py", line 377, in main
    lambda_ = n_offsprings, cxpb=cxpb, mutpb=mutpb, ngen=ngen, stats=stats, verbose=True, halloffame=halloffame)
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/deap/algorithms.py", line 304, in eaMuPlusLambda
    for ind, fit in zip(invalid_ind, fitnesses):
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/futures.py", line 102, in _mapGenerator
    for future in _waitAll(*futures):
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/futures.py", line 358, in _waitAll
    for f in _waitAny(future):
  File "/home/minh/keras-tf-venv3/local/lib/python3.5/site-packages/scoop/futures.py", line 321, in _waitAny
    raise future.exceptionValue
ValueError: could not broadcast input array from shape (304) into shape (358)
[2018-05-14 23:50:26,772] launcher  (127.0.0.1:41517) INFO    Root process is done.
[2018-05-14 23:50:26,777] launcher  (127.0.0.1:41517) INFO    Finished cleaning spawned subprocesses.
[2018-05-14 23:50:26,814] launcher  INFO    SCOOP 0.7 1.1 on linux using Python 3.5.2 (default, Nov 23 2017, 16:37:01) [GCC 5.4.0 20160609], API: 1013
[2018-05-14 23:50:26,814] launcher  INFO    Deploying 8 worker(s) over 1 host(s).
[2018-05-14 23:50:26,814] launcher  INFO    Worker distribution: 
[2018-05-14 23:50:26,814] launcher  INFO       127.0.0.1:	7 + origin
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[Errno 2] No such file or directory: './configs/esn/Rainfall_bayeridge_pca.json'
File: ./esn_experiment.py
In path: /home/minh/PycharmProjects/MultivariateESN
[2018-05-14 23:50:27,352] launcher  (127.0.0.1:39501) INFO    Root process is done.
[2018-05-14 23:50:27,352] launcher  (127.0.0.1:39501) INFO    Finished cleaning spawned subprocesses.
